userguide/supervised-learning/linear-regression.md

```diff
@@ -115,8 +115,8 @@ shows the largest positive and negative coefficients. After a model is
 created, we can access the coefficients as follows:
 
 ```python
-coefs = model['coefficients']
-print(coefs)
+coefs = model.coefficients
+coefs
```

Comment:
I think you probably still want the `print` and the parentheses. Without that, the below output will not be displayed unless you're in interactive mode of python/ipython.

Reply:
Thanks - added back the print() statement